scripts/dev/prepare-delivery-probe.py

```
@@ -0,0 +1,120 @@
+        action="store_true",
+        help="удалить кеш, чтобы движка заведомо не было",
+    )
+    arguments = parser.parse_args()
+
+    target = host_target()
+    tag, assets = release_assets(arguments.tag)
+    engine_asset = f"{arguments.engine}-runtime-{target}.tar.gz"
+    core_asset = f"unica-runtime-{target}.tar.gz"
+
+    print(f"релиз {tag}, цель {target}")
+    print(f"  ядро:  {core_asset}: {'есть' if core_asset in assets else 'НЕТ'}")
```

Ruff (0.16.1)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)